src/commands/initProjectForVSCode/initProjectForVSCode.ts

```diff
@@ -43,10 +43,11 @@ export async function initProjectForVSCode(context: IActionContext, fsPath?: str
     }
 
     language = language || getGlobalSetting(projectLanguageSetting) || await detectProjectLanguage(context, projectPath);
+    const languageModel: number | undefined = getGlobalSetting(projectLanguageModelSetting) || await detectProjectLanguageModel(language, projectPath);
```

Comment:
Global setting refers specifically to user settings, i.e. workspace/workspace folder settings are ignored. Does it make sense for a language model setting to be a user setting?

Reply:
The purpose of this logic is to "VS Code"-ify an existing project, so the assumption is that there likely wouldn't be a local VS Code settings file.  Hence, most of the settings are inferred from non-VS Code content (like for the project language) if they're not set at the global level.